src/routes-data/sidebar-routes.jsx

```diff
@@ -0,0 +1,198 @@
+                label: 'Payments',
+            },
+            {
+                path: 'application',
+                element: <div>This is a child of random</div>,
+                label: 'Application',
+            },
+        ],
+    },
+];
+export const resources = [
+    {
+        path: 'resources',
+        label: 'Resources',
+        children: [
+            {
+                path: 'api-guide',
+                element: <ApiGuide />,
+                label: 'Api Guide',
+            },
+            {
+                path: 'faq',
+                element: <Faq />,
+                label: 'FAQ',
+            },
+            {
+                path: 'json-schemas',
+                element: <Json />,
+                label: 'JSON',
+            },
+            {
+                path: 'bug-bounty',
+                element: <BugBounty />,
+                label: 'Bug Bounty',
+            },
+        ],
+    },
+];
+
+// Stitching all the arrays together to one route object for the Router.
+export const sidebar_routes = [...getting_started, ...what_can_you_do, ...resources];
+
+const ImplementDropdown = () => {
+    const location = useLocation();
+    const [isActive, setIsActive] = useState(false);
+    const handleToggleDropdown = () => {
+        setIsActive(!isActive);
+    };
+    return (
+        <div className={styles.dropdown}>
+            <div className={styles.dropdownBtn} onClick={handleToggleDropdown}>
+                Implement Now
+                <span className={`${styles.arrow} ${isActive ? styles.down : ''}`} />
+            </div>
+            {isActive && (
+                <div className={styles.dropdownList}>
+                    {implement_now.map(items => (
+                        <Link
+                            key={items.path}
+                            to={"getting-started/implement-now/"+items.path}
+                            className={`${styles.dropdownContent} ${items.path === location.pathname.split('/')[4] ? styles.selected : ""}`}
+                        >
+                            {items.label}
+                        </Link>
+                    ))
+}
+                </div>
+            )}
+        </div>
+    );
+};
+
+function LinkComponent({ route, path }) {
+    const {pathname} = useLocation();
+   
+    return (
+        <div key={route.label}>
+            {route.children ? (
+                <div className={styles.menuHeader}>{route.label}</div>
+            ) : (
+                <Link to={path} className={`${styles.menuItem} ${path === pathname.replace('/docs/','') ? styles.selected : ''}`}>
+                    {route.label}
+                </Link>
+            )}
+            {route.children &&
+                route.children.map((child) => {
+                    // If there are children, recursively go over the nested children
+                    // till there are none anymore.
+                    return !child.is_collapsible ? (
+                        <>
```

Comment:
use `React.Fragment` to be more descriptive also following the standard as other projects.